rfcs/2022-04-09-accessibility.md

```diff
@@ -0,0 +1,167 @@
+  - Example: Using your phone one handedly in a crowded metro station
+
+<img src="./images/accessibility/ms-inclusive-design.png" alt="Microsoft Inclusive Design Shows Spectrum Of Disabilities" width="25%" />
+
+# Motivation
+- Why are we doing this?
+- What use cases does it support?
+- What is the expected outcome?
+
+## Why are we doing this? 
+
+Without proper accessibility, certain functionality used in web sites is not available to some users with disabilities, especially people who rely on screen readers and people who cannot use a mouse. 
+
+Making our products accessible is important for two reasons.  
+- it's the right thing to do. We believe that everyone deserves to be able to use our products, regardless of their abilities.
+- It's good for business & branding. By making our products accessible, we can expand our potential customer base and make our products more appealing to a wider range of people, plus it helps create a good faith between consumers and our product. 
+
+**Making customers happy & respecting human rights:**  
+
+In 2020, A 29 year old banker who is visually impaired opened petitions to make [zomato](https://www.change.org/p/zomato-make-zomato-accessible-with-screen-readers-for-visually-challenged-people) & [swiggy](https://www.change.org/p/swiggy-make-swiggy-accessible-with-screen-readers-in-android-for-visually-challenged-people) accessible because he couldn’t order food from it due to their inaccessible apps. 
+Over **15,000 & 27,000 people had signed the petition.**
+Later zomato, swiggy listened to the people and [improved their accessibility.](https://theprint.in/tech/why-the-visually-challenged-are-signing-petitions-addressed-to-apps-such-as-zomato-swiggy/507282/) 
+
+
+## The current state  
+
+In current state, blade components are not very accessible out of the box, neither are some of our production grade mobile applications. 
+
+Here’s the screen reader accessibility of Razorpay mobile app
+
+- Missing accessibilityLabel in input box.
+- Missing accessibilityRole=button in buttons
+- Missing accessibilityRole=link in links
+- Grouping text together so they get announced at once.
+- Unlabeled password toggle icon 
+- Wrongly focus on text input's placeholder text
+
+
+https://user-images.githubusercontent.com/35374649/167380507-71e60b7f-d36e-42b3-b649-911400e2c66f.mp4
+
+
+Even though only a few of the teams are on boarded to blade, when more teams will migrate this will become a problem since every app/site will be inaccessible if we don’t implement accessibility in blade components. 
+
+Adding accessible components to blade will iron out all these inconsistencies and teams can build more accessible apps without thinking about it. 
+
+## What is the expected outcome? 
+
+The goal of this RFC is to lay the foundations for implementing accessibility right into blade, so that blade consumers don’t have to think about accessibility while building their products: 
```

Comment:
```suggestion
The goal of this RFC is to lay the foundations so that we can ship blade components that are accessible out of the box such that consumers don't have to worry about every minute detail. Here are couple of areas that this RFC will touch upon
```